db/migrate/20230601025347_create_search_results.rb

```diff
@@ -0,0 +1,18 @@
+class CreateSearchResults < ActiveRecord::Migration[7.0]
+  def change
+    create_table :search_results do |t|
+      t.string :keyword
+      t.integer :status
+      t.integer :adwords_top_count
+      t.integer :adwords_total_count
+      t.text :adwords_top_urls
+      t.integer :non_adwords_count
+      t.text :non_adwords_urls
+      t.integer :total_links_count
+      t.text :html_code
+      t.string :search_engine
```

Comment:
Should we only add the fields that we're using in this PR?

We can always update the table later on  🚀

Reply:
@Tuubz 👌 Then we will need to clean the database to avoid invalid old records in the database.